default.mspec

```diff
@@ -15,14 +15,17 @@ class MSpecScript
   # Command line specs
   set :command_line, [ 'command_line' ]
 
+  # Vulnerability resistance specs
+  set :vulnerabilities, [ 'vulnerabilities' ]
```

Comment:
I am thinking to call this `security` specs.
Vulnerabilities is quite specific and long (so easy to mistype).
What do you think?